storybookjs/storybook
# Update mdx.md

Adds a recipe for creating a Changelog story, which seems like a pretty common MDX-only docs use case.

Issue:

## What I did

Added docs that codify this pattern for adding arbitrary markdown dynamically to an MDX file:

https://github.com/storybookjs/storybook/issues/7644#issuecomment-1087721229

## How to test

~- [ ] Is this testable with Jest or Chromatic screenshots?~
~- [ ] Does this need a new example in the kitchen sink apps?~
- [x ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
